### PR TITLE
cgcreate: add page

### PR DESCRIPTION
**Pull request by Ray6464**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/cgcreate.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# cgcreate
+
+> Create cgroups, used to limite, mesure, and control resources used by processes.
+> You can create cgroups with types like `memory`, `cpu`, `net_cls`, etc.
+> More information: <https://www.kernel.org/doc/Documentation/cgroup-v1/>.
+
+- To install use:
+
+`apt install cgroup-lite cgroup-tools`
+
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:10]: This completely depends on the Distro being used.
```suggestion
```

File: pages/linux/cgcreate.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- To create a new group use:
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:11]:
```suggestion
- Create a new group:
```

File: pages/linux/cgcreate.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`cgcreate -g {{groupType}}:{{groupName}}`
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:13]:
```suggestion
`cgcreate -g {{group_type}}:{{group_name}}`
```

File: pages/linux/cgcreate.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- To create a new group with multiple croup types:
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:15]:
```suggestion
- Create a new group with multiple cgroup types:
```

File: pages/linux/cgcreate.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`cgcreate -g {{groupType1}},{{groupType2}}:{{groupName}}`
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:17]:
```suggestion
`cgcreate -g {{group_type1}},{{group_type2}}:{{group_name}}`
```

File: pages/linux/cgcreate.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- To create subgroups:
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:19]:
```suggestion
- Create a subgroup:
```

File: pages/linux/cgcreate.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+`mkdir /sys/fs/cgroup/{{groupType}}/{{groupName}}/{{subgroupName}}`
```
Comment (marchersimon) [inline pages/linux/cgcreate.md:21]:
```suggestion
`mkdir /sys/fs/cgroup/{{group_type}}/{{group_name}}/{{subgroup_name}}`
```

File: pages/linux/cgcreate.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# cgcreate
+
+> Create cgroups, used to limite, mesure, and control resources used by processes.
+> cgroups types can be `memory`, `cpu`, `net_cls`, etc.
```
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/linux/cgcreate.md:4]: Where should I open an issue about this? `tldr-lint`, `tldr-bot` or here?

[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/cgcreate.md:4]: tldr-lint @marchersimon I think

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**